sdk/examples/client/output/build_account_output.rs

```diff
@@ -49,9 +49,9 @@ async fn main() -> Result<()> {
     // Account id needs to be null the first time
     let account_output = AccountOutputBuilder::new_with_minimum_amount(storage_score_params, AccountId::null())
         .add_feature(SenderFeature::new(address.clone()))
-        .add_feature(MetadataFeature::new(metadata)?)
+        .add_feature(MetadataFeature::build().with_key_value("data", metadata).finish()?)
         .add_immutable_feature(IssuerFeature::new(address.clone()))
-        .add_immutable_feature(MetadataFeature::new(metadata)?)
+        .add_immutable_feature(MetadataFeature::build().with_key_value("data", metadata).finish()?)
```

Comment:
What is different in this example that needs to use `build().with_key_value()` and not just `new`?

Reply:
I just sprinkled the alternate usage in the examples. I didn't feel like updating them all 🤷‍♀️

Reply:
Ok but as a general question, why is it better?

Reply:
It's just a better syntax imo, users don't have to deal with three levels of parentheses due to the array of tuples situation. looks cleaner and easier to understand what's happening

Reply:
Alright